examples/website/src/shared/route/articleAdapter.ts

```diff
@@ -10,6 +10,7 @@ export function peerAdapter(peer: any): Peer {
     logoURL: peer.profile.logo?.squareURL,
     websiteURL: peer.profile.websiteURL,
     themeColor: peer.profile.themeColor,
+    themefontColor: peer.profile.fontColor,
```

Comment:
is this a typo? Or hwy only `fontColor`?

Reply:
I thought that would be more precise and prevent confusion if we ever need to use font colors anywhere else in future.